src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/CacheUtils.java

```diff
@@ -91,16 +121,7 @@ public static boolean isPrivatelyCacheable(ArrayFW<HttpHeaderFW> response)
                 }
             }
         }
-        return response.anyMatch(h ->
-        {
-            final String name = h.name().asString();
-            final String value = h.value().asString();
-            if (STATUS.equals(name))
-            {
-                return CACHEABLE_BY_DEFAULT_STATUS_CODES.contains(value);
-            }
-            return false;
-        });
+        return true;
```

Comment:
Recommend single `return` with `loop: for (String directive : parser) { ... }` and `break loop;` instead.